apps/meteor/app/apps/server/bridges/users.ts

```diff
@@ -121,13 +121,6 @@ export class AppUserBridge extends UserBridge {
 			return true;
 		}
 
-		const { status } = fields;
-		delete fields.status;
```

Comment:
Is it necessary to remove the `status` field? It is valid in Rocket.Chat's `IUser` type def https://github.com/RocketChat/Rocket.Chat/blob/390523f3103134a767286420a15a5d02696cafc5/packages/core-typings/src/IUser.ts#L119-L140 - maybe it should be kept?

Reply:
the first one makes sense to me, I will be removing the `{ statusDefault: string }` but I don't get the second comment, I am removing all the lines since it is not relevant to remove the field status since it will be considered in the User update

Reply:
Previously I used the UserPresence.changeDefaultStatus, then @sampaiodiego change it to use the prop statusDefault, but I don't think this is the case now, we need to change the status not the defaultStatus